torch/_inductor/codecache.py

```diff
@@ -888,11 +888,8 @@ def compile(cls, graph, source_code, cuda):
 
                 cls.cache[key] = output_so
 
-        def wrapper_call(*args):
-            assert len(graph.graph_outputs) > 0
-            return cls.cache[key], *(None for i in range(len(graph.graph_outputs) - 1))
-
-        return wrapper_call
+            return cls.cache[key]
+        return None
```

Comment:
Why do we need these changes ?

Reply:
Instead of returning a function, @desertfire said we should just directly return the so_path, so this just gets rid of the wrapper.